test/cpp/api/rnn.cpp

```diff
@@ -552,6 +555,13 @@ TEST_F(RNNTest, BidirectionalLSTMReverseForward_CUDA) {
 }
 
 TEST_F(RNNTest, BidirectionalMultilayerGRU_CPU_vs_CUDA) {
+#ifdef USE_CUDA
+  // Get device properties
+  const auto prop = at::cuda::getCurrentDeviceProperties();
+  const auto tolerance = prop->major == 8 && prop->minor == 9 ? 2e-5 : 1e-5;
```

Comment:
is this really the way to go? so sm8.9 is 2E-5 because of fp16/fp8? Or what is the reason for the drop in accuracy?

Reply:
I'm not sure, but IMO it's OK to do for now and investigate cause of the failure later, I'll create an issue to track all skipped/xfailed tests that were added here to be investigated later.

I can skip the test if you think it's a better behavior.

Reply:
A comment explaining the different tolerance values could be helpful to future maintainers here

Reply:
No preference, we could also just change to 2e-5 across all hardware to simplify.

Reply:
Actually, seems to fail now with outcome 4E-5

Reply:
Yeah, I've increased it to 2e-4 and created issue to track those failures